lib/module/utils.js

```diff
@@ -20,10 +20,10 @@ function getOptions (options) {
     options.targetFormats = options.pictureFormats;
   }
   if ('maxIdleDuration' in options) {
-    consola.warn(`[${pkg.name}] Option \`maxIdleDuration\` is deprecated, use \`maxIdleDurations\` instead. \`https://nuxt-speedkit.grabarzundpartner.dev/options#maxidledurations\``);
-    options.maxIdleDurations = {
-      entry: options.maxIdleDuration, hydrate: 8
-    };
+    consola.warn(`[${pkg.name}] Option \`maxIdleDuration\` is deprecated, can be removed is no longer needed.`);
```

Comment:
Option \`maxIdleDuration\` is deprecated and not necessary anymore. Specification can be removed from the configuration.